app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java

```diff
@@ -119,25 +120,36 @@ public NotificationsAdapter(AdapterDataSource<NotificationViewData> dataSource,
                         .inflate(R.layout.item_status_placeholder, parent, false);
                 return new PlaceholderViewHolder(view);
             }
+            default:
+            case VIEW_TYPE_UNKNOWN: {
+                View view = new View(parent.getContext());
+                view.setLayoutParams(
+                        new ViewGroup.LayoutParams(
+                                ViewGroup.LayoutParams.MATCH_PARENT,
+                                Utils.convertDpToPx(parent.getContext(), 24)
+                        )
+                );
+                return new RecyclerView.ViewHolder(view) {};
```

Comment:
similar to `StatusBaseViewHolder.showContent`
https://github.com/tuskyapp/Tusky/blob/c3df44eaef589c98731e417e42c119a60169e0f8/app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java#L234

Reply:
Is it insisible then or?..